package.json

```diff
@@ -16,8 +16,13 @@
     "packages/*"
   ],
   "devDependencies": {
-    "@changesets/cli": "^2.27.1",
-    "turbo": "^1.13.2"
+    "@changesets/cli": "^2.27.9",
+    "turbo": "^2.3.0"
   },
-  "packageManager": "yarn@4.4.1"
+  "packageManager": "yarn@4.4.1",
```

Comment:
are you sure this is the right package.json file?

Reply:
I think so! I did a check in the yarn.lock file:

- `rollup` is and indirect dependency of `vite`
    - `yarn.lock` file, `tsup@npm:^6.6.3`, line 7110
- `cookie` is and indirect dependency of `@azure/functions`
    - `yarn.lock` file, `@azure/functions@npm:^4.0.1, @azure/functions@npm:^4.5.0, @azure/functions@npm:^4.6.0`, line 185
- `vite` is and indirect dependency of `vitest`
    - `yarn.lock` file, `@vitest/mocker@npm:2.1.5`, line 2699